docs/uninstall.md

```diff
@@ -0,0 +1,449 @@
+
+<!-- This indentation is important for things to render properly. -->
+{{< tabs name="eventing_brokers" default="MT-Channel-based" >}}
+{{% tab name="Apache Kafka Broker" %}}
+
+1. Uninstall the Kafka Broker data plane by running the following command:
+
+    ```bash
+    kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-broker.yaml" >}}
+    ```
+
+1. Uninstall the Kafka controller by running the following command:
+
+    ```bash
+    kubectl delete -f {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-controller.yaml" >}}
+    ```
+
+{{< /tab >}}
+
+{{% tab name="MT-Channel-based" %}}
+
+Uninstall the broker by running:
+
+```bash
+kubectl delete -f {{< artifact repo="eventing" file="mt-channel-broker.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+
+### Uninstalling an optional default Channel (messaging) layer
+
+If you installed a default Channel (messaging) layer, uninstall it:
+
+<!-- This indentation is important for things to render properly. -->
+
+   {{< tabs name="eventing_channels" default="In-Memory (standalone)" >}}
+   {{% tab name="Apache Kafka Channel" %}}
+
+Uninstall the Apache Kafka Channel by running:
+
+   ```bash
+   curl -L "{{< artifact org="knative-sandbox" repo="eventing-kafka" file="channel-consolidated.yaml" >}}" \
+    | sed 's/REPLACE_WITH_CLUSTER_URL/my-cluster-kafka-bootstrap.kafka:9092/' \
+    | kubectl delete -f -
+   ```
+
+   <!-- Ideally write steps for uninstalling Apache Kafka for Kubernetes here. -->
+
+{{< /tab >}}
+
+{{% tab name="Google Cloud Pub/Sub Channel" %}}
+
+Uninstall the Google Cloud Pub/Sub Channel by running:
+
+   ```bash
+   kubectl delete -f {{< artifact org="google" repo="knative-gcp" file="cloud-run-events.yaml" >}}
+   ```
+
+{{< /tab >}}
+
+{{% tab name="In-Memory (standalone)" %}}
+
+Uninstall the in-memory channel implementation by running:
+
+```bash
+kubectl delete -f {{< artifact repo="eventing" file="in-memory-channel.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="NATS Channel" %}}
+
+1. Uninstall the NATS Streaming channel by running:
+
+    ```bash
+    kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-natss" file="300-natss-channel.yaml" >}}
+    ```
+
+1. Uninstall NATS Streaming for Kubernetes. For more information, see the [eventing-natss](https://github.com/knative-sandbox/eventing-natss/tree/main/config) repository in GitHub.
+
+{{< /tab >}}
+
+<!-- TODO(https://github.com/knative/docs/issues/2153): Add more Channels here -->
+
+{{< /tabs >}}
+
+
+### Uninstalling the Eventing component
+
+1. Uninstall the Eventing core components by running:
+
+    ```
+    kubectl delete -f https://storage.googleapis.com/knative-nightly/eventing/latest/eventing-core.yaml
+    ```
+
+1. Uninstall the required custom resources by running:
+
+    ```
+    kubectl delete -f https://storage.googleapis.com/knative-nightly/eventing/latest/eventing-crds.yaml
+    ```
```

Comment:
Again, I'm not sure if you need both core and crds, I think core includes the CRDs.

Reply:
I'll leave as is for now then, given there's not certainty. They're separate files so I can see why one might need to delete both separately.